packages/foreman/rubygem-hammer_cli_foreman/rubygem-hammer_cli_foreman.spec

```diff
@@ -4,26 +4,24 @@
 %global gem_name hammer_cli_foreman
 %global confdir hammer
 
-%global release 1
-%global prerelease .pre.develop
-
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
 Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.19
-Release: %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
+Version: 0.19.0
+Release: 1
```

Comment:
you shouldn't drop `%{dist}` (the others are OK, but you could also remove the `prerelease` line above and that's be it)